crates/aranya-fast-channels/src/client.rs

```diff
@@ -53,14 +53,22 @@ impl<S: AfcState> Client<S> {
         v
     }
 
+    /// Set up the seal context for the given channel.
+    ///
+    /// This must only be called once for any given ID. Failure to
+    /// do so will result in repeated sequence numbers.
```

Comment:
```suggestion
    /// This must only be called once for any given channel ID. Failure to
    /// do so will result in repeated sequence numbers (a.k.a. a repeated nonce).
```
What is the consequence of a repeated nonce in this case? Does it leak the key or plaintext? Does it make us vulnerable to a chosen plaintext/ciphertext attack? May be worth documenting the risk here.